src/main/kotlin/no/nav/familie/ks/sak/common/util/Utils.kt

```diff
@@ -3,3 +3,8 @@ package no.nav.familie.ks.sak.common.util
 fun Any?.nullableTilString() = this?.toString() ?: ""
 
 fun String.storForbokstav() = this.lowercase().replaceFirstChar { it.uppercase() }
+
+inline fun <reified T : Enum<T>> konverterEnumsTilString(liste: List<T>) = liste.joinToString(separator = ";")
+
+inline fun <reified T : Enum<T>> konverterStringTilEnums(string: String?): List<T> =
+    if (string.isNullOrBlank()) emptyList() else string.split(";").map { enumValueOf(it) }
```

Comment:
Nice! 🙌

Reply:
Hadde det vært mulig med tester på disse to her? 🙌

Reply:
Ja, det er vel mulig! 👍